client/src/core/client/admin/App/Navigation/Navigation.tsx

```diff
@@ -31,6 +32,12 @@ const Navigation: FunctionComponent<Props> = (props) => (
         <NavigationLink to="/admin/dashboard">Dashboard</NavigationLink>
       </Localized>
     )}
+    {/* TODO: Any other permissions needed? */}
```

Comment:
Ultimately, I think this is a question for design, but I would imagine this will be tied to an org mod level role? I don't know if, say, a site mod will be allowed to process DSA reports.

Reply:
Yep, also what I think, just pending confirmation from design so I thought I'd leave the note in here so it's not forgotten.